auth_session_timeout/models/res_users.py

```diff
@@ -17,28 +17,70 @@
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
-    def _check_session_validity(self, db, uid, passwd):
+    def _auth_timeout_urls_get(self, session, db, uid):
+        # Pluggable method for calculating ignored urls
+        # Defaults to stored config param
+        param_obj = self.pool['ir.config_parameter']
```

Comment:
FYI, self.env is not defined in these methods.
Was able to use request.env, but I'd rather find out why self.env isn't working.